docfx/articles/threading.md

```diff
@@ -0,0 +1,51 @@
+# Threading
+It is common for a developer to run one or more background tasks on a separate
+`Thread` (or `Task`). For example to periodically fetch data. If you want to
```

Comment:
Please use  the "infinitive form" in API docs instead of the "impersonal you". The infinitive form of a verb is its basic, uninflected form, typically preceded by the word "to" in English. For example, "to eat," "to play," or "to learn." By using the infinitive form, the sentence becomes more concise and focuses on the action itself.

This would be rewritten as "To make changes to a `View` from a background task, issue the change on the main Terminal.Gui Thread thread."

Miguel tends to write using the "impersonal you" form, so a lot of the old docs still use this. But I've worked hard to rid most API docs of it. Thanks.